cranelift/entity/src/set.rs

```diff
@@ -71,11 +74,11 @@ where
     /// `clear`ed or created with `new`.
     pub fn cardinality(&self) -> usize {
         let mut n: usize = 0;
-        for byte_ix in 0..self.len / 8 {
+        for byte_ix in 0..self.len / BITS {
```

Comment:
I see a couple of places where variables have `byte` in their names, which is confusing now that they aren't actually bytes. Could you search for the word `byte` in this file and pick new names? In this case, I think `byte_ix` could be `idx` instead.

As future work, this loop could be replaced with something like `self.elems[..self.len / BITS].iter().map(|x| x.count_ones() as usize).sum()`. That should be faster by avoiding bounds checks. But I'd want that in a separate PR. I like that you've kept this PR simple, making only one change.